icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
But this is a global alias. It's not namespaced to the author app (which would render author templates). This means no other app or non-authors-app-template can use this alias. If this is an alias provided and used by the authors app, in authors app templates, we can use whatever general name we like, but namespace it to the authors plugin/app.